src/patterns/find-a-support-centre/examples/results/_support-centre.scss

```diff
@@ -0,0 +1,42 @@
+/* --------------------------------
+
+Title:
+Support centre
+
+Description:
+Support centre listing
+
+-------------------------------- */
+
+.support-centre {
+  margin-bottom: 0;
+  padding-left: 0;
+}
+
+.support-centre__item {
+  display: block;
+  margin-bottom: 0;
+
+  p:last-child {
+    margin-bottom: 0;
+  }
+}
+
+.support-centre__item:not(:first-child) {
+  border-top: 1px solid #d8d8d8;
```

Comment:
Can this colour be se using one of the variables in the styles

Reply:
Yes this can use the `$color-grey-3`. I will update this.